js/data/bucket.js

```diff
@@ -65,8 +61,8 @@ function Bucket(options) {
     this.zoom = options.zoom;
     this.overscaling = options.overscaling;
     this.layer = options.layer;
+    this.childLayers = options.childLayers;
```

Comment:
I'm not seeing the child/parent relationship but I agree that the one character difference could be confusing. This is fine as is